# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/doc/Debugger/Debugger.Script.md
# Commit: e590eb34efc5
# Full Hash: e590eb34efc52ab109f2737167cb2aa0975a1792
# Author: Iain Ireland <iireland@mozilla.com>
# Date: 2021-06-01 09:43:14
# Regressor Bug: 1554524
# File Overlap Count: 1
# Description:
#   Bug 1624792: Remove Debugger interface r=tcampbell
#   
#   This patch removes the changes from bug 1554524 part 4. (Parts 1 and 3 of bug 1554524 were bytecode emitter cleanups that don't need to be reverted; part 2 was abandoned.)
#   
#   Depends on D116308
# ==============================================================================

diff -r 135a0e6af641 -r e590eb34efc5 js/src/doc/Debugger/Debugger.Script.md
--- a/js/src/doc/Debugger/Debugger.Script.md	Tue Jun 01 01:31:21 2021 +0000
+++ b/js/src/doc/Debugger/Debugger.Script.md	Tue Jun 01 01:31:21 2021 +0000
@@ -390,13 +390,6 @@
 
 **If the instance refers to WebAssembly code**, throw a `TypeError`.
 
-### `setInstrumentationId(id)`:
-**If the instance refers to a `JSScript`**, set the value which will be
-supplied as the script's ID to instrumentation callbacks in the script's
-realm. See `Debugger.Object.setInstrumentation()`.
-
-**If the instance refers to WebAssembly code**, throw a `TypeError`.
-
 ### Deprecated Debugger.Script Prototype Functions
 
 The following functions have all been deprecated in favor of `getOffsetMetadata`,